app/routes.js

```diff
@@ -439,6 +439,10 @@ module.exports.bind = function (app) {
   account.get(stripe.addPspAccountDetails, permission('stripe-account-details:update'), restrictToStripeAccountContext, stripeSetupAddPspAccountDetailsController.get)
 
   futureAccountStrategy.get(webhooks.index, permission('webhooks:read'), webhooksController.listWebhooksPage)
+  futureAccountStrategy.get(webhooks.create, permission('webhooks:read'), webhooksController.createWebhookPage)
+  futureAccountStrategy.post(webhooks.create, permission('webhooks:read'), webhooksController.createWebhook)
+  futureAccountStrategy.get(webhooks.update, permission('webhooks:read'), webhooksController.updateWebhookPage)
+  futureAccountStrategy.post(webhooks.update, permission('webhooks:read'), webhooksController.updateWebhook)
```

Comment:
Do we need a different permission for these routes? As `read` implies read-only access?